src/modules/fancyzones/lib/FancyZonesData.cpp

```diff
@@ -445,9 +445,9 @@ bool FancyZonesData::SetAppLastZones(HWND window, const std::wstring& deviceId,
     std::unordered_map<DWORD, HWND> processIdToHandleMap{};
     processIdToHandleMap[processId] = window;
     FancyZonesDataTypes::AppZoneHistoryData data{ .processIdToHandleMap = processIdToHandleMap,
-                                                  .zoneSetUuid = zoneSetId,
-                                                  .deviceId = deviceId,
-                                                  .zoneIndexSet = zoneIndexSet };
+                                                    .zoneSetUuid = zoneSetId,
+                                                    .deviceId = deviceId,
+                                                    .zoneIndexSet = zoneIndexSet };
```

Comment:
Can you restore the original formatting?

Reply:
No clue why VS decided to reformat this part. But it is back to original!